common/lib/xmodule/xmodule/modulestore/mongo/base.py

```diff
@@ -1804,3 +1810,53 @@ def ensure_indexes(self):
 
         # To allow prioritizing draft vs published material
         self.collection.create_index('_id.revision')
+
+    # Some overrides that still need to be implemented by subclasses
+    def convert_to_draft(self, location, user_id):
+        raise NotImplementedError(
+            "{function_name} needs to be implemented in subclass".format(
+                function_name=sys._getframe().f_code.co_name
```

Comment:
Ok - will change them.

(As an aside, all these (non) implementations were necessary because the "class" line was touched causing pre-existing errors to be detected as "differences").